### PR TITLE
Add lang,runtime,release info to test name to be displayed in sponge

### PR DESCRIPTION
**Pull request by adelez**

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

File: tools/interop_matrix/run_interop_matrix_tests.py [hunk 1]
```diff
@@ -163,7 +163,8 @@ def run_tests_for_lang(lang, runtime, images):
     # Download the docker image before running each test case.
     subprocess.check_call(['gcloud', 'docker', '--', 'pull', image])
     _docker_images_cleanup.append(image)
-    job_spec_list = find_test_cases(lang,release)
+    suite_name = '%s__%s_%s' % (lang, runtime, release)
```
Comment (matt-kwong) [inline tools/interop_matrix/run_interop_matrix_tests.py]: This got changed from `'%s__%s %s'` to `'%s__%s_%s'`. Is this intentional?

Comment (adelez) [reply tools/interop_matrix/run_interop_matrix_tests.py]: Yes, it's to avoid any potential issues with ' ' in terms of rendering and searching.

**Comment by adelez** [comment]
Actually I'm going to combine this PR with a different one, because I noticed the other one depends on this one.

**Pull request closed without merging**